content-src/components/Search/Search.js

```diff
@@ -0,0 +1,24 @@
+const React = require("react");
+
+const Search = React.createClass({
+  getDefaultProps() {
+    return {
+      onSearch: function() {}
+    };
+  },
+  render() {
+    return (<div className="search-wrapper">
+      <span className="search-label fa fa-search" />
+      <input ref="input" placeholder="Search" required />
```

Comment:
Do we need/want an `onEnter` handler or something here so that users can hit enter/return in the text input field instead of needing to click the submit button?